draft-ietf-sedate-datetime-extended.md

```diff
@@ -470,6 +464,8 @@ the summer of 2022, it is equivalent to:
 In contrast to this elective use of a suffix tag,
 
     2022-07-08T00:14:07+01:00[!Europe/Paris]
+    2022-07-08T00:14:07Z[!u-ca=chinese,u-ca=japanese]
+    2022-07-08T00:14:07Z[u-ca=chinese,!u-ca=japanese]
```

Comment:
These aren't syntactically valid according to the ABNF — should it be
```
    2022-07-08T00:14:07Z[!u-ca=chinese][u-ca=japanese]
    2022-07-08T00:14:07Z[u-ca=chinese][!u-ca=japanese]
```

Reply:
Yes.